swarm_copy/bluenaas_models.py

```diff
@@ -0,0 +1,307 @@
+# generated by datamodel-codegen:
```

Comment:
Is this the whole thing? 

I would never manually make modifications because we will be regenerating it on a regular basis so any manual modifications will be overwritten anyway.

Reply:
Ok:) Not big:)

Resolving